Common/src/CConfig.cpp

```diff
@@ -3587,8 +3587,12 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     SU2_MPI::Error("Only STANDARD_AIR fluid model can be used with US Measurement System", CURRENT_FUNCTION);
   }
 
-  if (nemo && Kind_TransCoeffModel != WILKE ) {
-    SU2_MPI::Error("Only WILKE transport model is stable for the NEMO solver.", CURRENT_FUNCTION);
+  if (Kind_FluidModel == SU2_NONEQ && Kind_TransCoeffModel != WILKE ) {
```

Comment:
```suggestion
  if (Kind_FluidModel == SU2_NONEQ && Kind_TransCoeffModel != TRANSCOEFFMODEL::WILKE ) {
```
You then need to change the type of Kind_TransCoeffModel from `unsigned short` to TRANSCOEFFMODEL and go around the code fixing the compiler errors basically by adding TRANSCOEFFMODEL::
@TobiKattmann which one is the `enum class` PR / discussion ?

Reply:
Starting of in #1220, going on in #1242 , then #1258 (and then in some other non-specific PR's) 👍 Note that if you `switch` and enum class, you must handle all cases (otherwise you get a warning) which can be using `default: break;` though

Reply:
Thanks. Exactly, which makes it very useful when introducing new enum cases to ensure they are handled everywhere.

Reply:
Thanks for the suggestion, I believe I managed to change the code to accommodate the enum class